modules/user_manual/pages/files/encrypting_files.adoc

```diff
@@ -87,7 +87,6 @@ Only the data in your files is encrypted, and not the filenames or
 folder structures. These files are never encrypted:
 
 * Old files in the trash bin.
-* Image thumbnails from the Gallery app.
```

Comment:
Image thumbnails are independent of the Gallery app. I think I wrote this in another PR already: Please just remove "from the Gallery app".

Question: Why is this same text written in three different files?

Reply:
I'll move the text to a separate, includable, partial.